docs/get-started/getting-started-kubernetes.md

```diff
@@ -263,5 +263,121 @@ sqlContext.sql("SELECT * FROM df WHERE value<>1").show()
 sc.stop()
 ```
 
+## Running Spark Applications using Spark Operator
+
+Using Spark Operator is another way to submit Spark Applications into a Kubernetes Cluster.
+
+1. Copy Spark Application jars/files into the docker image when preparing docker image.
```

Comment:
so is there any difference between this and the regular instructions for k8s?  either way we make the jars available in the docker image and set the configs appropriately.  Is the docker image different or can you use the same one?

Reply:
Correct. as per the Spark Operator user guide : "A SparkApplication should set .spec.deployMode to cluster, as client is not currently implemented. The driver pod will then run spark-submit in client mode internally to run the driver program. "

Reply:
@tgravescs  i just added a new sentence
```
This is because currently only cluster deployment mode is supported by Spark Operator.
```

Reply:
I'm not sure that addresses my concern, I was thinking more along the lines of the docker image. Above we state exactly how to create it and here we just say copy it into the docker image. If the docker image is created the exact same way, perhaps we just reference that section.

My other thought was actually do we even want this here at all?  It seems like a standard spark operator way of doing things so if there is nothing special do we want to maintain our own instructions?

Reply:
I see. Actually if we modify the initial dockerfile to put the test.py inside, then yes, it is the same docker image as previous examples, no difference. 
Since users may not need that test.py for their real production image, so we did not COPY it in the previous example.

IMO, the extra steps for "Spark Operator"'s most important goal is to provide an YAML file example. 
Other that, I did not put detailed steps on how to install "Spark Operator" here, and just point it to the external links.

Reply:
ok so I think we should clarify the docker image part. The docker has preparing the docker image at the top so I take that applies to all sections, but here we say again copy the jars to it, but we have already done that. 

Perhaps we need to split this into separate sub-sections. Like submitting with spark cli, submitting with spark operator and those are sections after the docker image creation, which is prerequisite.

Reply:
I just added an line when preparing the docker image saying that 
```
You can modify the Dockerfile to copy your application into the docker image, i.e. test.py.
```

Then now I changed it to "locate" the application file/jar instead of COPY.
Here we assume the test.py is the user's application file which is already copied into the docker image.